leapfrog_epla_natural.xml.fdm_material

```diff
@@ -0,0 +1,40 @@
+<?xml version='1.0' encoding='utf-8'?>
+<fdmmaterial version="1.3" xmlns="http://www.ultimaker.com/material" xmlns:cura="http://www.ultimaker.com/cura">
+  <metadata>
+    <name>
+      <brand>Leapfrog</brand>
+      <material>EPLA</material>
+      <color>Natural</color>
+    </name>
+    <description>Fast, safe and reliable printing. PLA is ideal for the fast and reliable printing of parts and prototypes with a great surface quality.</description>
+    <GUID>0ff92885-617b-4144-a03c-9989872454bc</GUID>
+    <compatible>True</compatible>
+    <version>3</version>
+    <adhesion_info>EPLA can be printed on a BuildTak sticker. Make sure the sticker is clean and free of debris. DO NOT use heavy duty solvents, such as acetone, to clean it, nothing stronger than rubbing alcohol.</adhesion_info>
+    <instruction_link>https://www.lpfrg.com/products/engineering-pla-filament</instruction_link>
+    <definition>fdmprinter</definition>
+    <color_code>#E9E6E6</color_code>
+  </metadata>
+  <properties>
+    <density>1.24</density>
+    <diameter>1.75</diameter>
+  </properties>
+  <settings>
+    <setting key="heated bed temperature">40.0</setting>
+    <setting key="print temperature">210</setting>
+    <setting key="standby temperature">210</setting>
+    <setting key="surface energy">100</setting>
+    <machine>
+            <machine_identifier manufacturer="Leapfrog B.V." product="Leapfrog Bolt Pro"/>2
+            <hotend id="Brass 0.4">
+                <setting key="hardware compatible">yes</setting>
+            </hotend>
+            <hotend id="NozzleX 0.4">
+                <setting key="hardware compatible">yes</setting>
+            </hotend>
+            <hotend id="Copper 0.4">

```

Comment:
This variant was not provided with the pullrequest of https://github.com/Ultimaker/Cura/pull/6608. Did you forget it?

Reply:
We have removed this hot end type for now since it is not yet being used. 

Also we have adjusted PVA to be it's natural color.